requirement-spi/src/main/java/org/arquillian/cube/openshift/impl/requirement/RequiresOpenshift.java

```diff
@@ -4,11 +4,12 @@
 import java.lang.annotation.Retention;
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
+import org.arquillian.cube.spi.requirement.Requirement;
 import org.arquillian.cube.spi.requirement.Requires;
 
 @Retention(RetentionPolicy.RUNTIME)
 @Target({ElementType.TYPE, ElementType.METHOD})
-@Requires(OpenshiftRequirement.class)
+@Requires(Requirement.class)
```

Comment:
This looks a bit odd now. WDYT @lordofthejars?

Reply:
Yeah really odd.

Reply:
Can you suggest me something using which we can make it clean

Reply:
I am really bad given names hehehe maybe `constraint`?

Reply:
I don't think it's even about the name, but the general construct. For now, maybe renaming is enough, but we might want to revisit our approach with annotations. Something tells me that we might need to improve the design